integration_tests/e2e/refer.cy.ts

```diff
@@ -392,19 +394,14 @@ context('Refer', () => {
           .build({ courseId: courses[0].id, prisonNumber: person.prisonNumber })
 
         cy.task('stubCreateParticipation', courseParticipation)
+        cy.task('stubParticipation', courseParticipation)
```

Comment:
Is it worth encapsulating the button click and this stub into a submit method, like on the `checkAnswers`, `confirmOasys` and `reason` pages?

Reply:
Yeah, like it